### PR TITLE
[OKTA-730760] Content issues in guide

### PR DESCRIPTION
**Pull request by annejuan-okta**
## Description:
- **What's changed?**  Fixed link, broken section, and some capitalization
- **Is this PR related to a Monolith release?**  Nope

### Resolves:

* [OKTA-730760](https://oktainc.atlassian.net/browse/OKTA-730760)

### CODE REVIEW

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/scim-with-entitlements/main/index.md](https://github.com/okta/okta-developer-docs/blob/66129f63450d5499751970792ea64c7d5ec55bc2/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fscim-with-entitlements%2Fmain%2Findex.md) | [83](https://okta.acrolinx.cloud/api/v1/checking/scorecards/5eacc32e-cf65-4485-81c8-1c70bf63ced7) | 88  | 88  | 70  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.f227bdac-0b86-4ef3-a616-b5ea10007f6f?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

File: packages/@okta/vuepress-site/docs/guides/scim-with-entitlements/main/index.md [hunk 1]
```diff
@@ -52,53 +52,51 @@ This combination of endpoints enables user schema discovery. Okta combines the i
...
-1. Optional. If your app requires custom Entitlement, Role, or User schema extensions, expose the following endpoint:
+    - Any resource-specific endpoints for entitlements (for example: `/Licenses`, `/Roles`, and so on)
+1. Optional. If your app requires custom entitlement, role, or user schema extensions, expose the following endpoint:
     - `/Schemas`
 1. Configure your app in Okta.
 
 ### User operations
 
-Create, update, and import User operations are handled using [SCIM User operations](/docs/reference/scim/scim-20/#scim-user-operations). The User object should include any Entitlements, Roles, and custom User schema extensions that your app requires. See [User with entitlements and roles](#user-with-entitlements-and-roles) for an example.
+Create, update, and import user operations are handled using [SCIM User operations](/docs/reference/scim/scim-20/#scim-user-operations). The user object should include any Entitlements, Roles, and custom user schema extensions that your app requires. See [User with entitlements and roles](#user-with-entitlements-and-roles) for an example.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/scim-with-entitlements/main/index.md:71]:
```suggestion
Create, update, and import user operations are handled using [SCIM User operations](/docs/reference/scim/scim-20/#scim-user-operations). The user object should include any entitlements, roles, and custom user schema extensions that your app requires. See [User with entitlements and roles](#user-with-entitlements-and-roles) for an example.
```

File: packages/@okta/vuepress-site/docs/guides/scim-with-entitlements/main/index.md [hunk 2]
```diff
@@ -52,53 +52,51 @@ This combination of endpoints enables user schema discovery. Okta combines the i
...
-- Roles
+- `Users`
+- `Groups`
+- `Entitlements`
+- `Roles`
 
 Okta doesn't offer custom handling for groups.
 
-To expose Entitlements and Roles, you must create a corresponding `ResourceType` for each of these entities. See [ResourceTypes](#resourcetype) for an implementation example.
+To expose entitlements and roles, you must create a corresponding `ResourceType` for each of these entities. See [ResourceTypes](#resourcetypes) for an implementation example.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/scim-with-entitlements/main/index.md:88]:
```suggestion
To expose entitlements and roles, create a corresponding `ResourceType` for each of these entities. See [ResourceTypes](#resourcetypes) for an implementation example.
```

File: packages/@okta/vuepress-site/docs/guides/scim-with-entitlements/main/index.md [hunk 3]
```diff
@@ -116,11 +114,11 @@ Each resource includes the following fields:
...
-* Schema extensions: List any extensions required for other properties. Generally, entitlements and roles don't need to have extensions, while it's common for Users to have highly customized extensions. For example, a user might have a schema extension to store a custom attribute for a particular app.
+* Schema extensions: List any extensions required for other properties. Generally, entitlements and roles don't need to have extensions, while it's common for users to have highly customized extensions. For example, a user might have a schema extension to store a custom attribute for a particular app.
 
-The default location for ResourceTypes definitions is `BaseURL/scim/v2/ResourceTypes`. For example, if your server is hosted on [https://example.com](https://example.com) then the available ResourceTypes can be retrieved from `https://example.com/scim/v2/ResourceTypes`.
+The default location for `ResourceTypes` definitions is `BaseURL/scim/v2/ResourceTypes`. For example, if your server is hosted on `https://example.com`, then the available ResourceTypes can be retrieved from `https://example.com/scim/v2/ResourceTypes`.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/scim-with-entitlements/main/index.md:119]:
```suggestion
The default location for `ResourceTypes` definitions is `BaseURL/scim/v2/ResourceTypes`. For example, if your server is hosted on `https://example.com`, then you can retrieve the available `ResourceTypes` from `https://example.com/scim/v2/ResourceTypes`.
```

**Review by susanharper-okta — COMMENTED** [review]
couple edits

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/scim-with-entitlements/main/index.md](https://github.com/okta/okta-developer-docs/blob/eb1c176ae7810c52f32d6f2eec4fd0de1bfc8ee0/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fscim-with-entitlements%2Fmain%2Findex.md) | [83](https://okta.acrolinx.cloud/api/v1/checking/scorecards/20cf0faa-e049-4230-91d5-641d58785c6a) | 88  | 88  | 70  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.cde48215-ff11-4c55-a92a-e6d5ef059072?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/scim-with-entitlements/main/index.md](https://github.com/okta/okta-developer-docs/blob/0a3f4f5316fb9dc1e14938ee774ac35c4c7c9e31/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fscim-with-entitlements%2Fmain%2Findex.md) | [83](https://okta.acrolinx.cloud/api/v1/checking/scorecards/338fd7a4-502a-4612-93a3-aea638eb59fc) | 88  | 88  | 70  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.d72ecc5c-c303-431c-97e0-73e440c56631?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/scim-with-entitlements/main/index.md](https://github.com/okta/okta-developer-docs/blob/2f4e0a5c9c90ac42ee2b79cf62b6347866cdf58d/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fscim-with-entitlements%2Fmain%2Findex.md) | [83](https://okta.acrolinx.cloud/api/v1/checking/scorecards/54ba7b83-64f1-4239-a51a-3950febd033d) | 88  | 88  | 70  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.08c947e9-53ea-4e88-af35-bf9bae188764?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Review by susanharper-okta — APPROVED** [review]
LGTM

**Pull request merged**